django/core/validators.py

```diff
@@ -128,7 +134,7 @@ def __call__(self, value):
                 raise
         else:
             # Now verify IPv6 in the netloc part
-            host_match = re.search(r'^\[(.+)\](?::\d{2,5})?$', urlsplit(value).netloc)
+            host_match = re.search(r'^\[(.+)\](?::\d{2,5})?$', self.split_url(value).netloc)
```

Comment:
ah right, I'll see if they actually are and update the PR accordingly. Thanks for the hint!